.github/workflows/main.yml

```diff
@@ -1,23 +1,15 @@
-  
 name: CI
 
-on: 
-  push:
-    branches:
-      - develop
-      - master
-  pull_request:
-    branches:
-      - master
-      - develop
+on: [push, pull_request]
 
 jobs:
   build:
     runs-on: ${{ matrix.os }}
     strategy:
+      fail-fast: false
       matrix:
         os: [ ubuntu-latest ]
-        smalltalk: [ Squeak64-5.2 ]
+        smalltalk: [ Squeak64-Trunk, Squeak64-5.3, Squeak64-5.2 ]
```

Comment:
Our tests sometime failed IIRC when several CI's were running at the same time. No idea if we ever fixed it, but this was the reason for only one CI image

Reply:
Yeah, I have experienced this issue, too, and fixed it via 8170a51.